scripts/target.sh

```diff
@@ -5,39 +5,43 @@ set -e
 DEFAULT_SOCK="/tmp/jam_target.sock"
 
 # Target configuration using associative array with dot notation
+# Architecture-specific files: use format "arch:filename" for multiple architectures
 declare -A TARGETS
 
 # === VINWOLF ===
 TARGETS[vinwolf.repo]="bloppan/conformance_testing"
-TARGETS[vinwolf.cmd]="./linux/tiny/x86_64/vinwolf-target --fuzz $DEFAULT_SOCK"
+TARGETS[vinwolf.cmd.linux]="./linux/tiny/x86_64/vinwolf-target --fuzz $DEFAULT_SOCK"
 
 # === JAMZIG ===
 TARGETS[jamzig.repo]="jamzig/conformance-releases"
-TARGETS[jamzig.cmd]="./tiny/linux/x86_64/jam_conformance_target -vv --socket $DEFAULT_SOCK"
+TARGETS[jamzig.cmd.linux]="./tiny/linux/x86_64/jam_conformance_target -vv --socket $DEFAULT_SOCK"
+TARGETS[jamzig.cmd.macos]="./tiny/macos/aarch64/jam_conformance_target -vv --socket $DEFAULT_SOCK"
 
 # === JAMDUNA ===
 TARGETS[jamduna.repo]="jam-duna/jamtestnet"
-TARGETS[jamduna.file]="duna_target_linux"
-TARGETS[jamduna.cmd]="./duna_target_linux -socket $DEFAULT_SOCK"
+TARGETS[jamduna.file]="linux:duna_target_linux macos:duna_target_mac"
```

Comment:
I prefer the same strategy as for cmd?
i.e 
```
TARGETS[jamduna.file.linux]=...
TARGETS[jamduna.file.macos]=...
```

then you pick the os where you need it